apps/fe-web/src/app/components/Search/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import styles from './Search.module.scss';
+
+export default function Search() {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <div className={styles.search}>
+      <div className={styles.inputWrapper}>
+        <input type='text' placeholder='Search' />
+      </div>
+      <button
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
```

Comment:
Remove this logic and is hovered state, use styles approach

`&:hover {
    background-color: $tertiary;
  }`